newm-server/src/main/resources/application.conf

```diff
@@ -270,11 +270,12 @@ mintingNotifications {
          messageUrl = "/email/mint-released.html"
          messageUrl = ${?EMAIL_MINT_RELEASED_MESSAGE_URL}
     }
-    passwordChanged {
-         subject = "Your password has been reset /Password reset confirmation"
-         messageUrl = "/email/password-changed.html"
-         messageUrl = ${?EMAIL_PASSWORD_CHANGED_MESSAGE_URL}
-    }
+}
+
+passwordChangeNotification {
```

Comment:
Why this part was taken out of the block?

Reply:
The block `mintingNotifications` should only encapsulate params for notifications related to minting (like the name suggests). Password reset is an unrelated category. 
Before this change, the original issue was that we're reading param as `passwordChanged.subject` instead of `mintingNotifications.passwordChanged.subject. Now with this chnage we must read it as `passwordChangeNotification.subject`

Reply:
Good catch ! Thanks for explaining